pyproject.toml

```diff
@@ -76,9 +77,11 @@ rocm624 = [
     "pytorch-triton-rocm>=2.0.0",
 ]
 
-deepsort = [
-    "scipy>=1.13.1",
+reid = [
+    "safetensors>=0.5.3",
+    "tensorboard>=2.19.0",
     "timm>=1.0.15",
+    "wandb>=0.19.11",
 ]
```

Comment:
If we go this route, `tensorboard` and `wandb` will become de facto mandatory requirements for using ReID with trackers. What about users who just want to use a pre-trained or fine-tuned feature extraction model? 

RF-DETR solves this by using a separate dependency extra:

```toml
metrics = [
    "tensorboard>=2.13.0",
    "wandb"
]
```

I think we can do something similar.